src/mensa_preis_handler/price_query_intent_handler.py

```diff
@@ -14,11 +14,23 @@
 class PriceQueryIntentHandler(AbstractRequestHandler):
     """Handler for PriceQueryIntent."""
 
-    def activate_session(self, db_collection_session):
-        db_collection_session.update_one({"SESSION_ACTIVE": False}, {"$set": {"SESSION_ACTIVE": True}})
+    def activate_session(self, db_collection_session, user_id):
+        # if this user has a session document
+        if db_collection_session.count_documents({"USER_ID": user_id}) > 0:
+            # set the SESSION_ACTICE to true, where the USER_ID is user_id
+            db_collection_session.update_one({"USER_ID": user_id}, {"$set": {"SESSION_ACTIVE": True}})
 
-    def deactivate_session(self, db_collection_session):
-        db_collection_session.update_one({"SESSION_ACTIVE": True}, {"$set": {"SESSION_ACTIVE": False}})
+        # otherwise create a session document
+        else:
+            session_doc = {
+                "USER_ID": user_id,
+                "SESSION_ACTIVE": True
+            }
+            db_collection_session.insert_one(session_doc)
+
+    def deactivate_session(self, db_collection_session, user_id):
+        # set the SESSION_ACTIVE to false, where the USER_ID is user_id
```

Comment:
Was bedeutet "USER_ID is user_id"?

Reply:
Wenn das Attribut USER_ID (auf der Datenbank) == user_id (die lokale Variable) ist.